doc/operator-application-capabilities.md

```diff
@@ -558,6 +558,12 @@ spec:
 
 * **NOTE 1**: `httpMethod`, `pattern`, `increment` and `metricMethodRef` fields are required.
 * **NOTE 2**: `metricMethodRef` holds a reference to the existing metric or method map key name `system_name`. In the example, `hits`.
+* **NOTE 3**: The combination of `pattern + httpMethod + metricMethodRef` must be unique among all mapping rules.
+  - If Product CR contains mapping rules where `pattern and httpMethod` are same, even if `metricMethodRef` fields differ -
```

Comment:
This should not be the case. If patter and httpMethod are same, but metric is different, the metric should not be a duplicated metric following 3scale.

Reply:
From what it looks here:
```
		if desired.Pattern == existingRule.Pattern &&
			desired.HTTPMethod == existingRule.HTTPMethod &&
			desiredMetricID == existingRule.MetricID 
```
The above sentence isn't true?